setup.py

```diff
@@ -22,6 +22,13 @@
     ]
 
 
+extras_require = {
+    'tests': tests_require,
+    'deletepermission': [
+        'collective.deletepermission',
```

Comment:
Why?

We usually have it either like this or move the extras-dict into the `setup` function call.
The reason why the `tests_require`-list is defined outside of the `setup` function call is that the list is used twice, once as `tests_require`-argument and once as value for `tests` in the `extras_require` argument dict.